src/core/thread/child_mask.hpp

```diff
@@ -58,78 +54,8 @@ namespace ot {
  * This class represents a bit-vector of child mask.
  *
  */
-class ChildMask
+class ChildMask : public BitVector<OPENTHREAD_CONFIG_MLE_MAX_CHILDREN>
```

Comment:
How about using `typedef` here?
```
typedef BitVector<OPENTHREAD_CONFIG_MLE_MAX_CHILDREN> ChildMask;
```

We can make it a sub-class later if/when we have more APIs that we want to add to `ChildMask`?